2019-11-13 23:23:29.608 - [Info] MITM Version: 1.26
2019-11-13 23:23:29.615 - [Info] Auto Access Enabled: true
2019-11-13 23:23:29.615 - [Debug] [Init] Auto Access Barrier: true
2019-11-13 23:23:29.636 - [Info] SSH man-in-the-middle server for 172.20.0.2 listening on 0.0.0.0:10101
2019-11-13 23:37:22.271 - [Debug] [Connection] Attacker connected: 178.128.247.219 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-13 23:37:22.932 - [Debug] [Auth] Attacker 178.128.247.219 trying to authenticate with "password"
2019-11-13 23:37:22.943 - [Debug] [Auto Access] Attacker: 178.128.247.219, Threshold: 1, Attempts: 1
2019-11-13 23:37:22.943 - [Debug] [Auto Access] Compromising the honeypot
2019-11-13 23:37:22.943 - [Debug] [Auto Access] Adding the following credentials: "root:xdhx123445"
2019-11-13 23:37:23.680 - [Debug] [Auth] Valid credentials - Password Authentication
2019-11-13 23:37:23.681 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.2
2019-11-13 23:37:23.841 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2019-11-13 23:37:24.256 - [Debug] [Connection] Attacker connected: 178.128.247.219 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-13 23:37:24.923 - [Debug] [Auth] Attacker 178.128.247.219 trying to authenticate with "password"
2019-11-13 23:37:24.923 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2019-11-13 23:37:24.940 - [Debug] [Auth] Attacker: 178.128.247.219 has so far made 1 attempts. Remaining: 5 attempts
2019-11-13 23:37:25.170 - [Debug] [Connection] Attacker closed the connection
2019-11-13 23:37:25.359 - [Debug] [EXEC] Noninteractive mode attacker command: cat /proc/cpuinfo | grep name | wc -l
